src/tests/compositor/Compositor_Kill_001.test.js

```diff
@@ -0,0 +1,65 @@
+import {
+  pluginDeactivate,
+  pluginActivate,
+  killClient,
+  getControllerPluginData,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'Compositor Kill - 001',
+  description: 'Kills the Plugin and validates the result',
+  steps: [
+    {
+      description: 'Deactivate Netflix Plugin and check deactivated or not',
+      test: pluginDeactivate,
+      params: constants.netFlixPlugin,
```

Comment:
lets use webkit instead of netflix please

Reply:
Updated code as per review comment